apps/website/.vuepress/config.js

```diff
@@ -22,8 +22,10 @@ module.exports = {
   dest: '../../dist/website',
   themeConfig: {
     algolia: {
-      apiKey: '88df2445b3dc4bf5e8bc4caafa9fbc50',
-      indexName: 'next_clarity',
+      // apiKey: '88df2445b3dc4bf5e8bc4caafa9fbc50', // https://next.clarity.design index
```

Comment:
@gnomeontherun - can you double check the apiKey and index name with the info you have from Algolia?